Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -404,13 +404,15 @@ public void ExternalReferences()
         {
             Deserialize<ExternalReferences>("17_ExternalReferences.xml", externalReferences =>
             {
+                var pathSeparator = Path.DirectorySeparatorChar.ToString();
+
                 Assert.NotNull(externalReferences.Texture);
-                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(@"\Xml\grass.tga"));
+                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(pathSeparator + "Xml" + pathSeparator + "grass.tga"));
```

Comment:
Can we just make these forward slashes in the xml file?  I don't think that would make the test worse and it should work on both Windows and non-Windows platforms.